inst/rmarkdown/templates/isba/resources/template.tex

```diff
@@ -0,0 +1,163 @@
+$endif$
+\usepackage{calc} % for calculating minipage widths
+% Correct order of tables after \paragraph or \subparagraph
+\usepackage{etoolbox}
+\makeatletter
+\patchcmd\longtable{\par}{\if@noskipsec\mbox{}\fi\par}{}{}
+\makeatother
+% Allow footnotes in longtable head/foot
+\IfFileExists{footnotehyper.sty}{\usepackage{footnotehyper}}{\usepackage{footnote}}
+\makesavenoteenv{longtable}
+$endif$
+
+$if(csl-refs)$
+% Pandoc citation processing
+\newlength{\cslhangindent}
+\setlength{\cslhangindent}{1.5em}
+\newlength{\csllabelwidth}
+\setlength{\csllabelwidth}{3em}
+\newlength{\cslentryspacingunit} % times entry-spacing
+\setlength{\cslentryspacingunit}{\parskip}
+% for Pandoc 2.8 to 2.10.1
+\newenvironment{cslreferences}%
+  {$if(csl-hanging-indent)$\setlength{\parindent}{0pt}%
+  \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces$endif$}%
+  {\par}
+% For Pandoc 2.11+
+\newenvironment{CSLReferences}[2] % #1 hanging-ident, #2 entry spacing
+ {% don't indent paragraphs
+  \setlength{\parindent}{0pt}
+  % turn on hanging indent if param 1 is 1
+  \ifodd #1
+  \let\oldpar\par
+  \def\par{\hangindent=\cslhangindent\oldpar}
+  \fi
+  % set entry spacing
+  \setlength{\parskip}{#2\cslentryspacingunit}
+ }%
+ {}
+\usepackage{calc}
+\newcommand{\CSLBlock}[1]{#1\hfill\break}
+\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\csllabelwidth}{#1}}
+\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \csllabelwidth}{#1}\break}
+\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}
+$endif$
+
+% Pandoc header
+$for(header-includes)$
+$header-includes$
+$endfor$
+
+$preamble$
+
+\begin{document}
+\begin{frontmatter}
+
+\title{$title$}
+\runtitle{$title$}
+
+\begin{aug}
+$for(author/pairs)$
+\author{$author.value.firstname$ $author.value.lastname$\thanksref{$author.value.affiliationref$$if(author.value.footnoterefs)$,$for(author.value.footnoterefs)$$it$$sep$,$endfor$$endif$}$if(author.value.email)$\ead[label=ea-$author.key$,email]{$author.value.email$}$endif$$if(author.value.url)$\ead[label=ua-$author.key$,url]{$author.value.url$}$endif$}$sep$, $endfor$
+
+\runauthor{$for(author/first)$$it.lastname$$endfor$$if(author/rest/rest)$ et al.$else$$for(author/rest)$ \and $author.lastname$$endfor$$endif$}
+
+$for(affiliations)$ %loop through affiliations
+\address[$affiliations.ref$]{$affiliations.name$, $affiliations.address$ $for(author/pairs)$
+$if(it.value.email)$
+\ifstrequal{$affiliations.ref$}{$it.value.affiliationref$}{\printead{ea-$it.key$}}{}
+$endif$
+$if(it.value.url)$
+\ifstrequal{$affiliations.ref$}{$it.value.affiliationref$}{\printead{ua-$it.key$}}{}
+$endif$
+$sep$ $endfor$%authors
+}$endfor$%affiliations
+
+$for(footnotes)$
+\thankstext{$it.ref$}{$it.text$}
+$endfor$
+
+\end{aug}
+
+\begin{abstract}
+$abstract$
+\end{abstract}
+
+% MSC class
+$if(MSC2020primary)$ \begin{keyword}[class=MSC]
+$for(MSC2020primary/first)$\kwd[Primary ]{$it$}$endfor$
+$if(MSC2020primary/rest)$$for(MSC2020primary/rest)$\kwd{$it$}$sep $$endfor$$endif$
```

Comment:
I think using only `for` should work - I don't think the `if` is needed here, is it ? 
Like if there is no `MSC2020primary/rest`, then it will do nothing

This would simplify the template

```suggestion
$for(MSC2020primary/rest)$\kwd{$it$}$sep $$endfor$
```

Reply:
Sure.